runtime/server_test.go

```diff
@@ -95,6 +99,17 @@ func TestDataV1(t *testing.T) {
                 "Message": "storage error (code: 1): bad path: [testmod p], path refers to non-array document with element p"
             }`},
 		}},
+		{"get with global", []tr{
+			tr{"PUT", "/policies/test", testMod, 200, ""},
+			tr{"GET", "/data/testmod/g?global=req1%3A%7B%22a%22%3A%5B1%5D%7D&global=req2%3A%7B%22b%22%3A%5B0%2C1%5D%7D", "", 200, "true"},
+		}},
```

Comment:
Does the query API allow us to POST global values?  Would be good to have that so people can run these queries using curl without having to url-encode the values.  (Or does curl have a mode for doing that automatically?)